README.rst

```diff
@@ -688,6 +688,22 @@ Remove all repositories:
       system:
         purge_repos: true
 
+Debian keyserver proxy setup globally:
+
+.. code-block:: yaml
+
+    linux:
+      system:
+        ...
+        proxy:
+          # By default, key download and keyserver requests do not use the
+          # global or per-repo proxies, but the Salt minion proxy.
+          # When present, 'linux:system:proxy:keyserver:http(s)' will be used
+          # as 'http(s)_proxy' for all `apt-key`-related operations.
+          keyserver:
```

Comment:
I would consider using system:proxy:pkg tree instead of using another place for defining another proxy.
Also there's per-repository proxy setup (see readme), I don't understand why to use one proxy for key retrieval and different proxy for anything else.

Reply:
This I forgot to justify.
When using MaaS to provision baremetal nodes, it already creates a global APT proxy entry with a name that contains 'curtin' in it (as it's handled via curtin templates). So I really don't want to add a duplicate proxy entry via system:proxy:pkg, nor do I need per-repo proxies in this scenario.
I admit I crafted this for my particular use-case and dismissed the idea of using system:proxy:pkg rather quickly. I'll respin this next week and try to address both comments.
Thank you for taking the time to review this!